src/camvidlog2/queries.py

```
@@ -66,3 +69,34 @@ def calculate_distances(
+    # still keep the column around so it can be referred to later
+    query_max.reset_index(names=["filename", "frame_no"], level=[1], inplace=True)
+    query_max.sort_values(
+        by="distance",
+        ascending=False,
+        inplace=True,
+    )
+
+    results = [
+        Result(rank=i, score=distance, filename=filename, frame_no=frame_no)
+        for i, (filename, frame_no, distance) in enumerate(
+            islice(query_max.itertuples(), num), 1
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)